annotated.go

```diff
@@ -1475,7 +1526,8 @@ type annotated struct {
 	Annotations []Annotation
 	ParamTags   []string
 	ResultTags  []string
-	As          [][]reflect.Type
+	As          [][]asType
+	AsSelf      bool
```

Comment:
Is this field needed? I don't see this used anywhere in this PR.

Reply:
Thanks for catching that!